src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -300,6 +326,8 @@ private bool CData()
                 Accept(_bufferedOpenAngle);
                 Accept(solidus);
```

Comment:
nit: while here please change this variable to something more widely known e.g. `slash`